bindings/java/CMakeLists.txt

```diff
@@ -39,40 +66,17 @@ set(JAVA_BINDING_SRCS
   src/main/com/apple/foundationdb/FutureStrings.java
   src/main/com/apple/foundationdb/FutureVoid.java
   src/main/com/apple/foundationdb/JNIUtil.java
-  src/main/com/apple/foundationdb/KeySelector.java
-  src/main/com/apple/foundationdb/KeyValue.java
   src/main/com/apple/foundationdb/LocalityUtil.java
   src/main/com/apple/foundationdb/NativeFuture.java
   src/main/com/apple/foundationdb/NativeObjectWrapper.java
-  src/main/com/apple/foundationdb/OptionConsumer.java
-  src/main/com/apple/foundationdb/OptionsSet.java
-  src/main/com/apple/foundationdb/package-info.java
```

Comment:
I think moving stuff would break a lot of folks (including wavefront) since we do probe the internal classes (various historical reasons). If there's one place for package-info.java it would probably be in api.

Reply:
Hm, well, I guess this is above my pay grade (and I take it you would be hesitant to make the change yourself that I've suggested given that it would break Wavefront code), but it would seem like any inspecting the internal classes is essentially depending on code outside of our official contract, i.e., clients are "at their own risk" if they rely on it. So, I guess even if it isn't changed in this PR, it does seem like something we might change in the future.

But as much as I'm not super excited about having the package split across two different projects, I guess it's probably okay if it does so. And I agree that the `package-info` should live in the API package if it can't live in both.